src/components/v5/common/ActionSidebar/partials/MultiSigSidebar/partials/MultiSigWidget/partials/ThresholdPassedBanner/ThresholdPassedBanner.tsx

```diff
@@ -0,0 +1,172 @@
+import { type ColonyRole } from '@colony/colony-js';
+import React, { type FC } from 'react';
+import { defineMessages } from 'react-intl';
+
+import ExternalLink from '~shared/ExternalLink/ExternalLink.tsx';
+import { type MultiSigUserSignature } from '~types/graphql.ts';
+import { type Threshold } from '~types/multisig.ts';
+import { getMultipleRolesText } from '~utils/colonyRoles.ts';
+import { formatText } from '~utils/intl.ts';
+
+import { useRelevantUserRoles } from '../../../../hooks/useRelevantUserRoles.ts';
+
+const displayName =
+  'v5.common.ActionSidebar.partials.MultiSig.partials.MultiSigWidget.partials.ThresholdPassedBanner';
+
+interface ThresholdPassedBannerProps {
+  approvalsPerRole: Record<number, MultiSigUserSignature[]>;
+  rejectionsPerRole: Record<number, MultiSigUserSignature[]>;
+  thresholdPerRole: Threshold;
+  requiredRoles: ColonyRole[];
+  multiSigDomainId: number;
+}
+
+const MSG = defineMessages({
+  cantApproveText: {
+    id: `${displayName}.cantApproveText`,
+    defaultMessage:
+      'The {passedRoles} {numberOfPassedRoles, plural, one {permission has} other {permissions have}} passed the required threshold. Only the {remainingRoles} {numberOfRemainingRoles, plural, one {permission is} other {permissions are}} required to approve. Your approval won’t count towards the total. ',
+  },
+  cantRejectText: {
+    id: `${displayName}.cantRejectText`,
+    defaultMessage:
+      'The {passedRoles} {numberOfPassedRoles, plural, one {permission has} other {permissions have}} passed the required threshold. Only the {remainingRoles} {numberOfRemainingRoles, plural, one {permission is} other {permissions are}} required to reject. Your rejection won’t count towards the total. ',
+  },
+  learnMoreLink: {
+    id: `${displayName}.learnMoreLink`,
+    defaultMessage: 'Learn more',
+  },
+});
+
+const MULTI_SIG_HELP_URL =
```

Comment:
Shouldn't this go in https://github.com/JoinColony/colonyCDapp/blob/64562f3423dd8a54c58907e3fed89889b50d4968/src/constants/externalUrls.ts ?